onnx/defs/tensor/defs.cc

```diff
@@ -765,10 +765,14 @@ ONNX_OPERATOR_SET_SCHEMA(
           }
         }));
 
-static const char* Gather_ver1_doc = R"DOC(
+static const char* Gather_ver10_doc = R"DOC(
 Given `data` tensor of rank r >= 1, and `indices` tensor of rank q, gather
 entries of the axis dimension of `data` (by default outer-most one as axis=0) indexed by `indices`, and concatenates
-them in an output tensor of rank q + (r - 1).
+them in an output tensor.
+If elem_index is disabled (default),  the indices represent the indices of the slices in the axis chosen, and the 
+output is of rank q + (r - 1).
+Otherwise the indices represent the indices of the elements in the tensor in the chosen axis, and the output shape is
+the same as the input shape.
```

Comment:
Should this be the "the output shape is the same as the indices shape"?

Reply:
yes I will update this